### ISSUE TITLE
miner.js offline?

### ISSUE THREAD

**Issue by fabriek-huis** [Kukunin/webminer#2]
https://minecrunch.co/web/miner.js - is offline where do i get the source? thanks

**Comment by Kukunin** [comment]
Yes, miner is offline. Sources of miner itself are closed, at least for now. There is public service like CoinHive where my miner is used

**Comment by fabriek-huis** [comment]
@Kukunin CoinHive is your project? or only they have access to your miner? I don't want to use connive i want to use a open source miner. Please let me know once they are public/online would love to see the source, Thanks

**Comment by Kukunin** [comment]
CoinHive is not my project, but we have a similar one. 
Currently we don't have plans to open the sources, but there is a plan to publish them so everyone can build own infrastructure around the miner
